packages/ui-modal/src/Modal/index.js

```diff
@@ -336,28 +337,21 @@ class Modal extends Component {
         {this.renderChildren()}
       </Dialog>
     )
-
-    if (size === 'fullscreen') {
-      return (
-        <span
-          className={classnames({
-            [styles.fullscreenLayout]: true,
-            [styles[`fullscreenLayout--${constrain}`]]: true
-          })}
-        >
-          {dialog}
-        </span>
-      )
-    } else {
-      return (
-        <Mask
-          placement={this.maskPlacement}
-          fullscreen={constrain === 'window'}
-        >
-          {dialog}
-        </Mask>
-      )
-    }
+    return (
+      <Mask
+        placement={this.maskPlacement}
+        fullscreen={constrain === 'window'}
+        theme={
+          isFullScreen ? { borderRadius: '0em', borderWidth: '0em' } : null
```

Comment:
this is to remove the 1px border when set to fullscreen to keep a pixel-by-pixel similarity